### ISSUE TITLE
[CHANGE] Training

### ISSUE THREAD

**Issue by mgifford** [gc-da11yn/gc-da11yn.github.io#213]
2.3.8.1 Training and tools for building in accessibility to ICT from the start must be available for public servants;
https://a11y.canada.ca/en/standards/standard/#2.3.8.1

Yes..  But why limit this to the public sector? EdX has a great course available online for free:
https://www.edx.org/course/web-accessibility-introduction

There are other great courses here too:
https://github.com/mgifford/a11y-courses

The Ontario Government has funded some amazing open source (Creative Commons):
https://www.torontomu.ca/accessibility/news-events/2019/03/web-accessibility-for-developers-ebook/

There is frankly amazing content available online for free that the Government of Canada should be using and contributing to. Events like https://a11yto.com/ can be a critical strategy for training employees. So can  locall groups like https://a11yyow.ca/ 

Champions networks are also key to changing the culture. Training will be forgotten if it isn't built into the culture of an organization https://abilitynet.org.uk/news-blogs/14-tips-build-accessibility-champions-network
